Runtime/Core/BucketManagement/BucketStore.cs

```diff
@@ -25,10 +26,12 @@ public class BucketStore : IBucketStore
         private static readonly string s_s3BucketNamePattern = @"(?=^.{3,63}$)(?!^(\d+\.)+\d+$)(^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$)";
 
         private readonly IAmazonS3Wrapper _amazonS3Wrapper;
+        private readonly string _bucketPolicyPath;
```

Comment:
Does this need to be passed in or can this be an internal implementation detail to the BucketStore? I don't see this changing from consumer to consumer so seems like a good idea to keep this internal to this class

Reply:
I can update this if required, but to me it made more sense to pass through the value from the assembly where the file is located annd follow existing patterns.

Reply:
Since this is for the logging bucket I don't see this pattern changing per consumer. If this were to change it would change for all consumers so i would prefer we keep this logic within this wrapper

Reply:
Changed